recipes-core/packagegroups/packagegroup-ni-restoremode.bb

```diff
@@ -36,10 +36,12 @@ RDEPENDS_${PN}_append_x64 = "\
     efivar              \
     fw-printenv         \
     grub                \
+    grub-efi            \
```

Comment:
Why do we need both `grub` and `grub-efi` here? I was under the impression that `grub-efi` was a RAUC thing.

Reply:
I believe we are using EFI for current gen also. Unpacked dist-nilrt-grub-gateway ipk and the iso in there also efi stuff.

![image](https://user-images.githubusercontent.com/8194523/141358330-8c6338fb-0996-437d-b919-f071da0128fe.png)

Reply:
As for whether we need `grub`, I'm not sure. I can add this as a task to figure out in the recovery image work item.

Reply:
I find it very confusing that `grub-efi` isn't just a package split of `grub`, but uses the same source. But OK.